pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -62,58 +62,88 @@ func newGitHubScraper(
 	}
 }
 
-func (ghs *githubScraper) getPullRequests(
+func getTotalPrPages(
+	ghs *githubScraper,
 	ctx context.Context,
 	client graphql.Client,
-	repos []SearchNode,
+	repoName string,
+	owner string,
+	now pcommon.Timestamp,
+) (int, error) {
+	prOpenCount, err := ghs.getPrCount(ctx, client, repoName, ghs.cfg.GitHubOrg, []PullRequestState{PullRequestStateOpen})
+	if err != nil {
+		return 0, err
+	}
+	ghs.logger.Sugar().Debugf("open pull request count: %v for repo %v", prOpenCount, repoName)
+	ghs.mb.RecordGitRepositoryPullRequestCountDataPoint(now, int64(prOpenCount), repoName)
```

Comment:
I'm thinking the metric name needs to change based on the differentiation between open and merge pull requests.

Reply:
I also wonder if recording of the metric should be done within this function.

Reply:
I think it makes sense since we'd need to pass out the count values otherwise and at least to me it feels like all of it belongs together

Reply:
i'm still contemplating this, any additional thoughts on the metric names?

Reply:
I seperated them into two different parts for merged and open.  I think it makes sense especially when the naming was sort of ambiguous until you looked at the metric description itself